sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -174,7 +174,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>1.7.36</version> <!-- {x-version-update;org.slf4j:slf4j-api;external_dependency} -->
```

Comment:
This is a breaking change - what motivated it? We will probably need to find an alternative

Reply:
I went through the versions used by spark 3.5 [here](https://spark.apache.org/releases/spark-release-3-5-0.html) , but I will leave it as 1.7.36 then.

Reply:
I think we need to understand the differences better - a new major version usually means breaking changes - understanding where the breaking changes are seems important to make the right call.

Reply:
The major changes in the new slf4j was changing their static binder mechanism to a service loader mechanism as said [here](https://www.slf4j.org/faq.html#changesInVersion200). This doesn't seem to affect us, so think its fine to leave as old version.